class-parsely-recommended-widget.php

```diff
@@ -9,24 +34,32 @@ public function __construct() {
 		parent::__construct( 'Parsely_Recommended_Widget', 'Parsely Recommended Widget', $widget_options );
 	}
 
+	/**
+	 * This is the widget function
+	 *
+	 * @category   Function
+	 * @package    WordPress
+	 * @subpackage Parse.ly
+	 * @param array $args Widget Arguments.
+	 * @param array $instance Values saved to the db.
+	 */
 	public function widget( $args, $instance ) {
 		$title = apply_filters( 'widget_title', $instance['title'] );
 
 		$instance['display_options'] = ! empty( $instance['display_options'] ) ? $instance['display_options'] : array();
-		echo $args['before_widget'] . $args['before_title'] . $title . $args['after_title'];
+		echo esc_html( $args['before_widget'] . $args['before_title'] . $title . $args['after_title'] );
```

Comment:
Were you able to smoke-test the `/profile` and `/related` api calls in your local environment? They way to do so would be to enter "elevatedtoday.com" as your API key in the WordPress CMS settings page as well as the elevatedtoday.com API secret, enable the widget, and then check to see if it's rendering correctly with elevatedtoday posts and whether they API calls in the dev tools network tab look right

Reply:
Good eye! This broke the widget, but I've got a solution now (that would have helped me much earlier if I had known about it).  For future reference wp_kses() is helpful with the escaping function messages from the linter.